dsymbol/src/dsymbol/conversion/package.d

```diff
@@ -63,10 +66,14 @@ struct ScopeSymbolPair
 	{
 		typeid(DSymbol).destroy(symbol);
 		typeid(Scope).destroy(scope_);
+		foreach(ufcsSym; ufcsSymbols){
+			typeid(DSymbol).destroy(ufcsSym);
+		}
```

Comment:
actually now that I see this code, I realized this introduces a bug, since we don't actually own the symbols. What I rather wanted was destroying just the array / freeing the memory for the pointers, but I think we can just live without doing that since it's GC allocated anyway.

Can you remove the destroy here again? I think it would otherwise corrupt memory. (and add a comment to ensure we don't accidentally add it in the future again)